includes/collection/class-actors.php

```diff
@@ -702,9 +702,11 @@ public static function get_actor( $post ) {
 		$post = \get_post( $post );
 
 		if ( ! $post ) {
+			$text = \__( 'Actor not found', 'activitypub' );
+			self::add_error( $post->ID, $text );
```

Comment:
yeah, without a `$post`, this won't work. Let's just limit it to the `init_from_json()` case you added below.

Reply:
oh :D